author.bib

```diff
@@ -644,30 +699,37 @@ @string{ Misailovic_S
 @string{ Mitra_D              = "Mitra, Debasis" }
 @string{ Mitra_S              = "Mitra, Sayan" }
 @string{ Mitri_S              = "Mitri, Sara" }
+@string{ Miyazaki_F           = "Miyazaki, Fumio" }
 @string{ Moen_J               = "Moen, Jonas" }
 @string{ Mohan_Y              = "Mohan, Yogeswaran" }
 @string{ MonasterioHuelin_F   = "Monasterio-Huelin, Félix" }
 @string{ Mondada_F            = "Mondada, Francesco" }
+@string{ Montana_L            = "Montana, Luis" }
 @string{ Montanier_JM         = "Montanier, Jean-Marc" }
 @string{ Montavon_G           = "Montavon, Grégoire"}
 @string{ MontesDeOca_M        = "Montes de Oca, Marco" }
 @string{ Moore_AW             = "Moore, Andrew W." }
 @string{ Moore_JM             = "Moore, Jared M." }
 @string{ Morais_MRA           = "Morais, Marcos Ricardo A." }
 @string{ Moran_F              = "Morán, F." }
+@string{ Moreira_AP           = "Moreira, António Paulo" }
 @string{ Moreno_A             = "Moreno, A." }
 @string{ Morgan_N             = "Morgan, Nelson"}
 @string{ Moses_ME             = "Moses, Melanie E." }
+@string{ Moslinger_C          = "Moslinger, Christoph" }
 @string{ Mostaghim_S          = "Mostaghim, Sanaz" }
 @string{ Mouret_JB            = "Mouret, Jean-Baptiste" }
 @string{ Mukherjee_A          = "Mukherjee, Apratim" }
 @string{ Muller_J             = "Müller, Jörg" }
 @string{ Muller_KR            = "Müller, Klaus-Robert"}
+@string{ MunozMartinez_V      = "Muñoz-Martinez, Victor" }
```

Comment:
```suggestion
@string{ MunozMartinez_VF      = "Muñoz Martínez, Víctor Fernando" }
```

Reply:
I checked his papers and it is a bit inconsistent with the author name. I think we can keep them all to be comprehensive. Please double verify.